src/pysocha/build.py

```diff
@@ -136,9 +139,12 @@ def generate_blog_posts(config: dict):
                                              'total_pages': total_pages}))
 
     output_filename = os.path.join(output_dir, 'authors' + file_ext)
+    first_author = None
```

Comment:
From my understanding this variable is going to set the author of the site in the atom feed. You could add an override to the `config.py` so the site maintainer doesn't need to make a blog post to be listed, and then default to the first blog post author.

Reply:
If there's no authors, the for loop generating the blog won't work, so this is guaranteed to either be populated or not matter unless misconfigured, as far as I can tell.